requirements.txt

```diff
@@ -1,3 +1,4 @@
 langchain-core==0.3.9
 langchain-community==0.3.1
 google-cloud-spanner==3.49.1
+langchain-experimental==0.3.2
```

Comment:
Since this dep is only needed for the colab, please remove from here.

Reply:
Done.